libs/blocks/quiz/utils.js

```diff
@@ -9,9 +9,14 @@ const RESULTS_EP_NAME = 'results.json';
 let configPath; let quizKey; let analyticsType; let analyticsQuiz; let metaData;
 const { locale } = getConfig();
 
-const initConfigPath = (roolElm) => {
-  const link = roolElm.querySelector('.quiz > div > div > a');
-  const quizConfigPath = link?.text.toLowerCase();
+export const initConfigPath = (rootElm) => {
+  const link = rootElm.querySelector('.quiz > div > div > a');
+  const quizConfigPath = link?.text?.toLowerCase() || '';
```

Comment:
It's a preexisting condition, but reading quizconfigpath from the first arbitrary link we find is pretty fragile. We should use the quizurl value from the existing global metadata object. Also, since we have this existing parameter in the quiz block, I think the parameter name that we use in the URL to override it should be the same. We should use quizurl all the way through, rather than quizurl and quiz-data. 

<img width="551" alt="Screenshot 2023-07-19 at 9 37 01 AM" src="https://github.com/adobecom/milo/assets/25230829/e1aa0e27-f5b6-47d0-a070-2e2f5c3cc575">